simapp/simd/main.go

```diff
@@ -13,10 +14,10 @@ func main() {
 	rootCmd, _ := cmd.NewRootCmd()
 
 	if err := svrcmd.Execute(rootCmd, simapp.DefaultNodeHome); err != nil {
-		switch e := err.(type) {
-		case server.ErrorCode:
```

Comment:
Do you know which lint rule enforces this change?
I'm just curious, the previous code style(type switch) looks simpler to me.